### DISCUSSION TITLE
Improve installation summary

### DISCUSSION THREAD

**Discussion by joseivanlopez** [agama-project/agama#778] [Ideas]
The installation summary is intented to show an overview of the most relevant selected options of the system to install. It has to be short but enough to get an idea of the important details before clicking on install. This is how the summary looks now:

![Screenshot from 2023-09-28 11-55-13](https://github.com/openSUSE/agama/assets/1112304/b0b07c38-f7cf-4b79-ae82-9b15272c0ad1)

Some problems I see with the information in that page:

* Using sentences (e.g., in Localization, Storage or Users) makes difficult to see the relevant information at first sight. Moreover, sentences could be confusing for some users, and translations are not always 100% accurate.
* In storage, finding a good sentence when there are several disks involved would be tricky.
* In network, it would be nice to clearly see if WiFi and/or wired is configured.
* In localization, the selected keyboard layout will be added, so the sentence will be longer.

### Proposal

A possible improvement could be to present the key information in a more structured way, using sentences only when really needed. Also using icons could help. For example:

![Screenshot from 2023-09-28 11-54-02](https://github.com/openSUSE/agama/assets/1112304/e3100df5-b4d4-4fbf-8248-6063123872b5)

**Comment by shundhammer** [comment]
- I agree with using sentences making it more difficult to grasp. We are not writing a novel here. I'd go for telegram style.

- I also agree that showing what network card is being used. I don't know if that is possible, but if we can get that information, something like "Onboard 1 GBit/sec Intel wired network" or "Onboard 54 MBit/sec Intel Wifi" would be nice, in particular if there is also a non-onboard network adapter.

In general, I like your proposal better than what we had before.

**Comment by shundhammer** [comment]
The "Software" section is still very much a sore point, but that's well known. It's very unspecific right now what will be installed. But AFAIK that's WiP.

**Comment by imobachgs** [comment]
I like the proposal. However, about the network summary:

* Remember that you can have dozens of devices. For that reason, in the summary, we are displaying just the connected devices.
* Please, include the IP address as it might be important.

**Comment by imobachgs** [comment]
And about the root authentication, I would try to summarize both facts into a single sentence.

**Comment by dgdavid** [comment]
I like the general idea of improving the summary and avoiding long sentences. About using an icon per value... is not that easy. Let's find a way that allow us to use them when really useful and meaning, but not as a rule.

As said yesterday in a comment, I also miss to know that encryption is set in the storage section.

NP: without lines separating sections as it's shown in the mockup. In fact, we had them and we stop using (do not have the PR at hand by now).

**Reply by joseivanlopez** [reply]
Yes, lines are only for the mockup. The mockup is only for sharing the idea, no styles proposed at all.

About showing encryption, I am not sure how important it is. There are a lot details in storage, for example, if LVM is used, if a separate file system is created for some mount point, etc. For me, the crucial storage info to have in the overview is what disks are affected and what is going to happen with the current data. The rest are more specific details you can see by going to the storage section (IMHO).

**Reply by dgdavid** [reply]
I see... but it's a **password** set. From my perspective is an important thing I'd like to know. On the other hand, it's not a value automatically proposed, but we are not only displaying proposed values but those we consider important enough for being there.

That said, if it's only me... I can survive without it in the summary 🥲

**Comment by ancorgs** [comment]
Another note, for the record:

Currently, the content of all the selected disks for the VG and also the content of the disk selected as installation device are affected by the so-called "space policy" (the actions to make space).

The Storage section in the current overview page only informs about the installation device, so the user could not realize that the content of other devices is going to be affected too.

We have an specific card for that at Trello. https://trello.com/c/GyOOEw20/236-storage-show-all-used-devices-in-the-storage-overview-section

**Comment by ancorgs** [comment]
This discussion will become obsolete when we merge https://github.com/openSUSE/agama/pull/1202 (very likely today). Let's close it already.
